src/v2/Apps/IdentityVerification/Error.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, Link, Sans, Serif } from "@artsy/palette"
+import { AppContainer } from "v2/Apps/Components/AppContainer"
+import React from "react"
+import { Title as HeadTitle } from "react-head"
+
+export const Processing: React.FC = () => {
+  return (
+    <AppContainer>
+      <HeadTitle>Artsy | ID Verification</HeadTitle>
+
+      <Box px={[2, 3]} mb={6} mt={4}>
+        <Box mx="auto" width={[335, "80%"]} maxWidth="400px" textAlign="center">
+          <Serif size="6" color="black100">
```

Comment:
For all new text we should be using our new [`<Text />`](https://palette.artsy.net/tokens/typography/) component.

Reply:
I force-pushed a fix.